pkg/kdump/kdump-view.jsx

```diff
@@ -41,14 +41,69 @@ import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/compon
 import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 
 import { useDialogs, DialogsContext } from "dialogs.jsx";
+import { read_os_release } from "os-release.js";
 import { fmt_to_fragments } from 'utils.jsx';
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { FormHelper } from "cockpit-components-form-helper";
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { ModificationsExportDialog } from "cockpit-components-modifications.jsx";
 
 const _ = cockpit.gettext;
 
+const exportAnsibleTask = (settings, os_release) => {
+    const target = Object.keys(settings.targets)[0];
+    const targetSettings = settings.targets[target];
+    const kdump_core_collector = settings.core_collector;
+
+    let role_name = "rhel-system-roles";
+    if (os_release.NAME.includes('Fedora')) {
+        role_name = "linux-system-roles";
+    }
+
+    let ansible = `
+---
+# ${role_name} is also available via ansible-galaxy
```

Comment:
That wouldn't be correct for rhel-s-r? Also, this should be an URL.

Reply:
Neither for Fedora, so I have no idea why we mention it actually. Both CentOS, RHEL, Fedora have it. Other distributions are not supported by the ansible role like SUSE.

Reply:
For rhel-s-r? But what is the system you run ansible from that could be anything?

Reply:
But you get os-release from the system that runs cockpit, so it should be fine. Of course if you run the role from another system, you have to adjust it.

Reply:
What I meant is: "rhel-system-roles" is **not** available from Ansible Galaxy, only the upstream linux-s-r is.